pkg/pool-manager/pool_test.go

```diff
@@ -100,13 +103,7 @@ var _ = Describe("Pool", func() {
 
 	Describe("Pool Manager General Functions ", func() {
 		It("should create a pool manager", func() {
-			fakeClient := fake.NewSimpleClientset()
-			startPoolRangeEnv, err := net.ParseMAC("02:00:00:00:00:00")
-			Expect(err).ToNot(HaveOccurred())
-			endPoolRangeEnv, err := net.ParseMAC("02:FF:FF:FF:FF:FF")
-			Expect(err).ToNot(HaveOccurred())
-			_, err = NewPoolManager(fakeClient, startPoolRangeEnv, endPoolRangeEnv, false)
-			Expect(err).ToNot(HaveOccurred())
+			createPoolManager("02:00:00:00:00:00", "02:FF:FF:FF:FF:FF")
```

Comment:
could you please do this change in a separate commit?

Reply:
done